composer.json

```diff
@@ -10,6 +10,9 @@
     "require": {
         "php": "^7.1.3",
         "fideloper/proxy": "^4.0",
+        "laradium/laradium": "dev-master",
+        "laradium/laradium-content": "dev-master",
+        "laradium/laradium-permission": "dev-master",
```

Comment:
This package will be removed and replaced with https://github.com/spatie/laravel-permission